hapi/hedera-protobufs/services/basic_types.proto

```diff
@@ -1837,3 +1837,47 @@ message ConsensusTokenFeeScheduleAllowance {
      */
     uint64 amount_per_message = 5;
 }
+
+/**
+ * A unique, composite, identifier for a topic allowance.
+ *
+ * Each topic allowance SHALL be uniquely identified by a TopicAllowanceId.
+ * A TopicAllowanceId SHALL be recorded when created and MUST be provided in any transaction
+ * that would modify that topic allowance (such as a `consensusApproveAllowance`).
```

Comment:
Markdown collapses line breaks; so we _encourage_ adding break tags at the end of each requirement sentence (except the last).
```suggestion
 * Each topic allowance SHALL be uniquely identified by a TopicAllowanceId.<br/>
 * This identifier SHALL be recorded when created.<br/>
 * This identifier MUST be provided in any transaction that would modify
 * that identified topic allowance (such as a `consensusApproveAllowance`).
```